mlflow/models/evaluation/default_evaluator.py

```diff
@@ -1423,29 +1428,27 @@ def _check_args(self, metrics, eval_df):
                 failed_metrics.append(result)
 
         if len(failed_metrics) > 0:
-            output_column_name = self.predictions
             output_columns = (
                 [] if self.other_output_columns is None else list(self.other_output_columns.columns)
             )
             input_columns = list(self.X.copy_to_avoid_mutation().columns)
 
-            error_messages = []
-            for metric_name, param_names in failed_metrics:
-                error_messages.append(f"Metric '{metric_name}' requires the columns {param_names}")
-            error_message = "\n".join(error_messages)
-            raise MlflowException(
-                "Error: Metric calculation failed for the following metrics:\n"
-                f"{error_message}\n\n"
-                "Below are the existing column names for the input/output data:\n"
-                f"Input Columns: {input_columns}\n"
-                f"Output Columns: {output_columns}\n"
-                "Note that this does not include the output column: "
-                f"'{output_column_name}'\n\n"
-                f"To resolve this issue, you may want to map the missing column to an "
-                "existing column using the following configuration:\n"
-                f"evaluator_config={{'col_mapping': {{'<missing column name>': "
-                "'<existing column name>'}}\n"
-            )
+            error_messages = [
```

Comment:
Can we add a quick test case for this `error_messages` being rendered correctly since it seems like a complicated error message

Reply:
modified existing test